existential-vs-universal-quantification/article.md

```diff
@@ -0,0 +1,160 @@
+# Existential vs universal quantification
+
+## Introduction
```

Comment:
Would add some information for why the person reading it would be interested in the article in the front. 

After that, you can delve into what it is.